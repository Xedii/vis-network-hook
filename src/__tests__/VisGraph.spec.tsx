import * as React from "react";
import { render, fireEvent, screen } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import VisGraphProvicer, { VisGraph } from "./../index";
import { data } from "./data";

describe("Renders react-vis-hooks", () => {
  test("Render provider", async () => {
    const { asFragment } = render(
      <VisGraphProvicer graph={data}>
        <VisGraph />
      </VisGraphProvicer>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
