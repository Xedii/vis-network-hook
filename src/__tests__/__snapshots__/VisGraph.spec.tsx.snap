// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<DocumentFragment>
  <div
    style="width: 100%; height: 100%;"
  >
    <div
      class="vis-network"
      style="position: relative; overflow: hidden; user-select: none; webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); width: 100%; height: 100%;"
      tabindex="900"
    >
      <canvas
        height="0"
        style="position: relative; user-select: none; webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); width: 100%; height: 100%;"
        width="0"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`Renders react-vis-hooks Render provider 1`] = `
<DocumentFragment>
  <div
    style="width: 100%; height: 100%;"
  >
    <div
      class="vis-network"
      style="position: relative; overflow: hidden; user-select: none; webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); width: 100%; height: 100%;"
      tabindex="900"
    >
      <canvas
        height="0"
        style="position: relative; user-select: none; webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0); width: 100%; height: 100%;"
        width="0"
      />
    </div>
  </div>
</DocumentFragment>
`;
